support-frontend/app/services/HttpIdentityService.scala

```diff
@@ -58,12 +59,23 @@ object IdentityService {
 }
 
 case class GetUserTypeResponse(userType: String)
-
 object GetUserTypeResponse {
   implicit val readsGetUserTypeResponse: Reads[GetUserTypeResponse] = Json.reads[GetUserTypeResponse]
   implicit val getUserTypeEncoder: Encoder[GetUserTypeResponse] = deriveEncoder
 }
 
+case class CreateSignInTokenResponse(encryptedEmail: String)
+object CreateSignInTokenResponse {
+  implicit val readCreateSignInTokenResponse: Reads[CreateSignInTokenResponse] = Json.reads[CreateSignInTokenResponse]
+  implicit val createSignInTokenResponseEncoder: Encoder[CreateSignInTokenResponse] = deriveEncoder
```

Comment:
I _think_ this `createSignInTokenResponseEncoder` is unnecessary.
The `readCreateSignInTokenResponse` is used to deserialize the response in `createSignInToken`

Confusingly we're mixing two json libraries here, circe and play-json (that's not your fault)

Reply:
Done ✅